src/e2e/app/cli/repo-tests/src/use_cases/mod.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+mod test_delete_account_use_case;
+
+pub use test_delete_account_use_case::*;
```

Comment:
I got a bit confused, use case tests in e2e. Probably, for e2e level of worldview, use case does not exist, probably it's just account deletion function